.github/workflows/main.yml

```
@@ -94,9 +151,99 @@ jobs:
           COVERALLS_PARALLEL: true
           COVERALLS_SERVICE_NAME: github
 
+  test-notebooks:
+    name: Test Notebooks
+    needs: lint
+    if: |
+      contains(github.event.pull_request.labels.*.name, 'notebooks') ||
+      contains(github.event.pull_request.labels.*.name, 'approved') ||
+      (github.event.review.state == 'approved') ||
+      (github.event_name == 'push')
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+       python-version: ["3.9", "3.10", "3.11"]
```

@Zeitsperre Est-ce que c'est juste ça pour ajouter 3.12 ?
```suggestion
       python-version: ["3.9", "3.10", "3.11", "3.12"]
```